config/locales/en-US.yml

```diff
@@ -242,8 +242,9 @@
           html: Sent by <b>%{sent_by}</b>.
           text: Sent by %{sent_by}.
     kudos:
-      guest: A guest
-      guests: guests
+      guest: 
+        one: a guest
```

Comment:
I'm having a hard time figuring it out -- is this for the sentence "A guest left kudos on %{work}" or "%{usernames} and a guest left kudos on %{work}"? If it's for the first one, it should be "A guest," not "a guest."